cdk/visit/lambda_code/log_visit.py

```diff
@@ -140,6 +140,8 @@ def handler(request, context):
     try:
         # Get the username from the request body.
         username = json.loads(request["body"])["username"]
+        username = username.lower()
```

Comment:
This change will lower-case the usernames in the visit records, but we currently have mixed-case registrations in our database and don't have a limit on the case for future registrations. DynamoDB works like a hashmap, and we're using the username as the primary key for the registrations. If the user has a mixed-case registration, we'll always look up the lower-cased username in the DynamoDB table & find nothing. This change will therefore still result in duplicate registration emails to any students that register(ed) with a mixed-case username.

Can you perhaps control the case of the registrations?

Reply:
+1 I think we should focus on ensuring username is downcased in the new tables since we already plan on migrating. Added a comment here to make sure that happens https://github.com/clemsonMakerspace/unified-makerspace/pull/88/files